content/events/online-hackfest/2020-uiux/faq.adoc

```diff
@@ -0,0 +1,81 @@
+---
+layout: project
+title: "Jenkins UI/UX Hackfest. Frequently Asked Questions"
+sigId: "advocacy-and-outreach"
+tags:
+  - outreach-programs
+  - events
+  - hackathon
+  - community
+  - ux
+links:
+  gitter: jenkinsci/hackfest
+opengraph:
+  image: /images/post-images/jenkins-is-the-way/jenkins-is-the-way-hackfest-opengraph.png
+---
+
+== General questions
+
+=== Why UI/UX?
+
+Jenkins user experience is one of the most popular negative feedback we get from users. 
+There is a wide perception that the Jenkins Web UI is dated compared to other CI/CD projects.
+This is not something new, this topic was discussed multiple times at contributor summits and finally led to the creation of the Jenkins UX SIG which currently coordinates the Look&Feel improvements in Jenkins.
+Our user documentation is also quite difficult to use: complex navigation on jenkins.io, a lot of user documentation still being on Wiki, etc.
+All these problems create negative optics for the project and impact Jenkins adoption, and IMO there are many opportunities to improve it.
+There are also a lot of newcomer-friendly and small stories which could be handled by contributors.
+
+=== Why May 25-29?
+
+These dates are quite close, yes. We propose these dates based on the availability of event hosts.
+It also fits the Google Summer of Code timeline.
+The event happens on the last week of the community bonding, and hence it would give accepted students and newcomer mentors to get introduced with the wider Jenkins community.
```

Comment:
```suggestion
The event happens on the last week of the community bonding, and hence it would give accepted students and newcomer mentors time to be introduced to the wider Jenkins community.
```